bee-ternary/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.3.4-alpha - 2020-11-13
```

Comment:
You did not update the version in Cargo.toml ?

Reply:
Oops!